sound/soc/sof/Kconfig

```diff
@@ -117,8 +117,8 @@ config SND_SOC_SOF_DEBUG_VERBOSE_IPC
 	  if you are trying to debug IPC with the DSP firmware.
           If unsure select "N".
 
-config SND_SOC_SOF_DEBUG_FORCE_IPC_POSITION
-	bool "SOF force to use IPC for position update on SKL+"
+config SND_SOC_SOF_DEBUG_DISABLE_NOWAKEUP
+	bool "SOF to disable non-interrupt mode for position update on SKL+"
 	help
 	  This option force to handle stream position update IPCs and run pcm
 	  elapse to inform ALSA about that, on platforms (e.g. Intel SKL+) that
```

Comment:
Maybe update the longer description. Something like:
This option forces use of position update IPCs. IPC updates are used even on platforms where other mechanisms (e.g. HDAC DPIB/posbuf) are available to query hardware position. On platforms (e.g. Intel SKL-)...

Reply:
Sure, I can edit the description more.

Reply:
Thanks! Updated text looks good now.